Tests/Config/Settings.ps1

```diff
@@ -1,6 +1,15 @@
 # Copyright (c) Microsoft Corporation. All rights reserved.
 # Licensed under the MIT License.
 
+<#
+	.NOTES
+	This file is hashed when the module is created, and that hash is stored in
```

Comment:
I'd like to modify this note a bit.  Suggested wording:

```
<#
.NOTES
A hash of this file is stored in GitHubConfiguration.ps1 as TestConfigSettingsHash (and can be retrieved
by calling Get-GitHubConfiguration -Name TestConfigSettingsHash).  It is used when trying to detect
if the file has been updated with personal settings before unit tests are run locally (since unit tests
will otherwise fail because users do not have account permissions to access the accounts below).

If this file is being modified as part of an intended change to the official repo, please be sure to
additionally update the value of TestConfigSettingsHash in GitHubConfiguration.ps1 by running:
  
    . ./Helpers.ps1; Get-Sha512Hash -PlainText (Get-Content -Path ./Tests/Config/Settings.ps1 -Raw -Encoding Utf8)
#>
```

Reply:
No problem.